extensions/ql-vscode/src/view/results/ResultsApp.tsx

```diff
@@ -77,6 +78,10 @@ export function ResultsApp() {
     isExpectingResultsUpdate: true,
   });
 
+  const [userSettings, setUserSettings] = useState<UserSettings>({
+    shouldShowProvenance: false,
+  });
```

Comment:
Should there be a `defaultUserSettings` constant somewhere since it's shared between the two components?

Reply:
Done